tests/attr/test_kernel_shap.py

```diff
@@ -386,6 +387,7 @@ def _kernel_shap_test_assert(
             )
 
             if expected_coefs is not None:
+                set_all_random_seeds(1234)
```

Comment:
nit: Could also add this before line 374, so for each iteration of the loop, both attribution calls are consistent.

Reply:
I remember that we discussed it and I tried but it didn't pass the tests. I think that is probably because since the seeds haven't been initialized, for some tests the expected values are computed based on the next seed value and doesn't work when I reset it. In that case we need to change the expected values for some test cases.

Reply:
Interesting, having set_all_random_seeds in both places should make sure they are always consistent (both at 390 and 373). Was the issue when adding the line in both places or putting it only at 374? 

Putting it in both places should ensure that the result values are identical with either return_input_shape being True or False, so might help to reduce issues in the future if there are further changes. But otherwise, this change looks good!